### PR TITLE
github issue template suggests who to tag

### PR DESCRIPTION
**Pull request by sshleifer**
Fewer issues get lost if people tag the relevant developer. It also saves @LysandreJik time.
While he is out, I figured we could experiment with trying to nudge issue raisers to tag.

Here is the comment at the beginning of the "Bug Report" template:

Your issue will be replied to more quickly if you can figure out the right person to tag with @
 If you know how to use git blame, that is the easiest way, otherwise, here is a rough guide of who to tag.
 Please tag fewer than 3 people.
 
 albert, bert, GPT2, XLM: @LysandreJik 
 tokenizers: @mfuntowicz
 Trainer: @julien-c
 Speed and Memory Benchmarks: @patrickvonplaten
 Model Cards: @julien-c
 Translation: @sshleifer
 Summarization: @sshleifer
 TextGeneration: @TevenLeScao 
 examples/distillation: @VictorSanh
 nlp datasets: [different repo](https://github.com/huggingface/nlp)
 rust tokenizers: [different repo](https://github.com/huggingface/tokenizers)
 blenderbot: @mariamabarham
 Bart: @sshleifer
 T5: @patrickvonplaten
 Longformer/Reformer: @patrickvonplaten
 TransfoXL/XLNet: @TevenLeScao 
 examples/seq2seq: @sshleifer
 tensorflow: @jplu 

I wrote it in 3 minutes in case people hate this idea, so I am sure it is missing people!
Suggestions very much appreciated.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 1]
```diff
@@ -7,7 +7,26 @@ assignees: ''
...
-# 🐛 Bug
+<!-- Your issue will be replied to more quickly if you can figure out the right person to tag with @
+ If you know how to use git blame, that is the easiest way, otherwise, here is a rough guide of who to tag.
+ Please tag fewer than 3 people.
+ 
+ albert, bert, GPT2, XLM: @lysandre
+ tokenizers: @mfuntowicz
+ Trainer: @julienc
+ Speed and Memory Benchmarks: @patrickvonplaten
+ Model Cards: @julienc
```
Comment (julien-c) [inline .github/ISSUE_TEMPLATE/bug-report.md:18]:
```suggestion
 albert, bert, GPT2, XLM: @LysandreJik 
 tokenizers: @mfuntowicz
 Trainer: @julien-c
 Speed and Memory Benchmarks: @patrickvonplaten
 Model Cards: @julien-c
```

**Comment by patrickvonplaten** [comment]
I like it! Maybe we can also put in **big** that people should add their env information and not insert screenshots - this happens still quite often from what I see

**Comment by TevenLeScao** [comment]
@sshleifer I think I've committed instead of suggesting an edit, sorry, lemme know if it broke anything!

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 2]
```diff
@@ -7,7 +7,29 @@ assignees: ''
...
+ tokenizers: @mfuntowicz
+ Trainer: @julien-c
+ Speed and Memory Benchmarks: @patrickvonplaten
+ Model Cards: @julien-c
+ Translation: @sshleifer
+ Summarization: @sshleifer
+ TextGeneration: @TevenLeScao 
+ examples/distillation: @VictorSanh
+ nlp datasets: [different repo](https://github.com/huggingface/nlp)
+ rust tokenizers: [different repo](https://github.com/huggingface/tokenizers)
```
Comment (sshleifer) [inline .github/ISSUE_TEMPLATE/bug-report.md:24]:
```suggestion
TransfoXL/XLNet: @TevenLeScao
Text Generation: @TevenLeScao
```

**Comment by sshleifer** [comment]
ok to merge @julien-c ?

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 3]
```diff
@@ -7,14 +7,51 @@ assignees: ''
...
+ Text Generation: @TevenLeScao
+ blenderbot: @mariamabarham
+ Bart: @sshleifer
+ Marian: @sshleifer
+ T5: @patrickvonplaten
+ Longformer/Reformer: @patrickvonplaten
+ TransfoXL/XLNet: @TevenLeScao 
+ examples/seq2seq: @sshleifer
+ tensorflow: @jplu 
+ -->
```
Comment (sgugger) [inline .github/ISSUE_TEMPLATE/bug-report.md:49]:
```suggestion
documentation: @sgugger
 -->
```

**Comment by sgugger** [comment]
Like this too! Added myself for issues linked to the documentation, feel free to add more my way.

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 4]
```diff
@@ -7,14 +7,52 @@ assignees: ''
...
+- Using distributed or parallel set-up in script?:
+
+### Who can help
+<!-- Your issue will be replied to more quickly if you can figure out the right person to tag with @
+ If you know how to use git blame, that is the easiest way, otherwise, here is a rough guide of **who to tag**.
+ Please tag fewer than 3 people.
+ 
+ albert, bert, GPT2, XLM: @LysandreJik 
+ tokenizers: @mfuntowicz
+ Trainer: @julien-c
```
Comment (sgugger) [inline .github/ISSUE_TEMPLATE/bug-report.md:30]:
```suggestion
 Trainer: @sgugger
```

Comment (sgugger) [reply .github/ISSUE_TEMPLATE/bug-report.md:30]: Taking the lead on this one, as seen with @julien-c

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 5]
```diff
@@ -7,14 +7,52 @@ assignees: ''
...
+ Trainer: @sgugger
+ Speed and Memory Benchmarks: @patrickvonplaten
+ Model Cards: @julien-c
+ Translation: @sshleifer
+ Summarization: @sshleifer
+ TextGeneration: @TevenLeScao 
+ examples/distillation: @VictorSanh
+ nlp datasets: [different repo](https://github.com/huggingface/nlp)
+ rust tokenizers: [different repo](https://github.com/huggingface/tokenizers)
+ TransfoXL/XLNet: @TevenLeScao
```
Comment (LysandreJik) [inline .github/ISSUE_TEMPLATE/bug-report.md:39]:
```suggestion
```

Duplicate

**Review by LysandreJik — APPROVED** [review]
LGTM!

**Comment by stefan-it** [comment]
@sshleifer could you tag me for `examples/token-classification` 🤔

**Pull request merged**